docs/src/lib/approximations/overapproximate_expmap.md

```diff
@@ -0,0 +1,16 @@
+
+```@contents
+Pages = ["overapproximate_expmap.md"]
+Depth = 3
+```
+
+```@meta
+CurrentModule = LazySets.Approximations
+```
+
+# Overapproximation
+
+```@docs
+taylor_expmap_truncation
+taylor_expmap_remainder
```

Comment:
Maybe you need to add `LazySets.Approximations.` in front. Not sure why because you exported them. I am just guessing.

Reply:
Ah, good point. That helped me understand the problem. You need to add `IntervalMatrices` here:

https://github.com/JuliaReach/LazySets.jl/blob/49ac692ca81165ba9f92a5790ed93305d143b12b/docs/make.jl#L2